lang/zh.json

```
@@ -867,84 +867,84 @@
       "condition": "拜访冬之村"
     },
     "tartaric_abyss_uro": {
-      "name": "Someone Familiar",
-      "condition": "In the Tartaric Abyss, find Urotsuki's doppelganger"
+      "name": "熟悉的身影",
+      "condition": "在赤之奈落里找到另一个流浪子的身影"
     },
     "butterfly_lady": {
-      "name": "Butterfly Lady",
-      "description": "Eyes up here!",
-      "condition": "Enter the butterfly room in Twilight Park"
+      "name": "蝴蝶女士",
+      "description": "这边看！",
+      "condition": "进入月光庭院的蝴蝶屋"
     },
     "ice_cream": {
-      "name": "Triple Scoop Ice Cream",
+      "name": "三倍Ice Cream",
       "description": "Feeling like heaven~",
-      "condition": "Arrive at Ice Cream Dream"
+      "condition": "抵达冰淇淋梦"
     },
     "five_guardians": {
-      "name": "Five Guardians",
-      "description": "Her dream is still going on",
-      "condition": "Visit all 5 guardians of Dream Park and enter the Guardians' Temple"
+      "name": "五位守护者",
+      "description": "狂梦未散，流浪不止",
```

```suggestion
      "description": "HER DREAM IS STILL GOING ON",
```
与另一个徽章的简介不同，这里是直接引用了風化祠地上的符文。我觉得可以直接使用英文。